jujugui/static/gui/src/app/components/entity-details/content/readme/readme.js

```diff
@@ -93,18 +121,48 @@ YUI.add('entity-content-readme', function() {
     */
     _getReadmeCallback: function(error, data) {
       if (error) {
-        console.error(error); 
+        console.error(error);
         this.setState({readme: 'No readme.'});
       } else {
         var readme = data;
-        this.setState({readme: this.props.renderMarkdown(readme)});
+        const renderMarkdown = this.props.renderMarkdown;
+        let renderer = new renderMarkdown.Renderer();
+        renderer.heading = (text, level) => {
+          const escapedText = text.toLowerCase().replace(/[^\w]+/g, '-');
```

Comment:
That's true, although it's much nicer to be able to link to /apache2/#readme rather than apache2/#readme1 for the probably quite rare case that there are duplicate headings. I had a look at how other docs handle these headings and everyone seems to just leave it up to the docs authors to make sure there are no duplicates.